post-clone-scripts/install-semantic-release.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
 
 read -rp "Your git username: " GIT_USERNAME
-sed -i '' "s+khaosdoctor+$GIT_USERNAME+g" package.json
+sed -i "s+khaosdoctor+$GIT_USERNAME+g" package.json
 
 read -rp "Your repository name: " REPO_NAME
-sed -i '' "s+template-node-ts+$REPO_NAME+g" package.json
```

Comment:
This is distro specific, this change won't work on MacOS as per my test below

![image](https://user-images.githubusercontent.com/3200560/166155079-fd50ba21-63ee-43b9-8955-48a8ae1571a9.png)

Can you please add a check for `uname` being `Darwin`?